homeassistant/components/plex/config_flow.py

```diff
@@ -0,0 +1,359 @@
+"""Config flow for Plex."""
+import logging
+import plexapi.exceptions
+import requests.exceptions
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components.media_player import DOMAIN as MP_DOMAIN
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_TOKEN,
+    CONF_URL,
+    CONF_USERNAME,
+    CONF_PASSWORD,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import callback
+
+from .const import (
+    CONF_SERVER,
+    CONF_SERVER_IDENTIFIER,
+    CONF_ENABLE_MEDIA_PLAYER,
+    CONF_ENABLE_SENSOR,
+    CONF_USE_EPISODE_ART,
+    CONF_SHOW_ALL_CONTROLS,
+    CONF_REMOVE_UNAVAILABLE_CLIENTS,
+    CONF_CLIENT_REMOVE_INTERVAL,
+    DEFAULT_PORT,
+    DEFAULT_SSL,
+    DEFAULT_VERIFY_SSL,
+    DOMAIN,
+    PLEX_SERVER_CONFIG,
+)
+from .errors import ConfigNotReady, NoServersFound, ServerNotSpecified, TokenMissing
+from .server import setup_plex_server
+
+_LOGGER = logging.getLogger(__package__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class PlexFlowHandler(config_entries.ConfigFlow):
```

Comment:
```suggestion
class PlexFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
```

Reply:
This is how it was before the last commit (https://github.com/home-assistant/home-assistant/pull/26157/commits/3ddde18485e848e23264677120e74530e0bc58c3). Unfortunately the linter would mark the `DOMAIN` import as unnecessary when using the new syntax, so I reverted to the "old" way.